### ISSUE TITLE
Conversion for Web Documentation

### ISSUE THREAD

**Issue by gonsie** [flux-framework/rfc#221]
I'm going to start converting the files in this repo to rst to include them in the readthedocs site. I have a few questions:

- How are the files in this repo used/viewed?
- When/why is the makefile run? 
- Are there any special usage of asciidoc that you think I should know about?

I am assuming that typically these files are viewed directly on GitHub. My initial plan is to convert each `.adoc` file to `.rst`, which shouldn't change viewing directly (too much). Let me know if there are other uses to consider (such as the distribution of these docs as part of a release?)

**Comment by grondo** [comment]
I think you are right, the main (only?) use case is viewing these RFCs on GitHub.

The makefile is run by developers who would like to read PDF copies, which are slightly more legible than the GitHub rendered asciidoc.

I can't think of any special usage of asciidoc within the RFCs. We do use an include syntax, but this doesn't work on GitHub anyway.

**Comment by SteVwonder** [comment]
We also use the Makefile to run a validator on our YAML snippets.  This is run as a part of Travis CI.

**Comment by grondo** [comment]
Thanks for remembering that @SteVwonder!

**Comment by gonsie** [comment]
There are 4 outstanding PRs (from @SteVwonder @grondo @trws @chu11). I don't want to interrupt the changes they are working on when I do the conversion. 

How close are these PRs to being merged? 
Maybe don't start any new PR's until the conversion happens?

Preview the conversion in [gonsie/rfc@rst-conversion](https://github.com/gonsie/rfc/tree/rst-conversion)

**Comment by grondo** [comment]
Thanks for bringing this up!
I've removed my PR as I don't think it is relevant any longer.

Seems like we should be able to merge the rest immediately, even if they aren't perfect.

**Comment by gonsie** [comment]
with PR #232, I think this issue is closed.
